pkg/vere/io/http.c

```diff
@@ -2266,6 +2336,8 @@ u3_auto*
 u3_http_io_init(u3_pier* pir_u)
 {
   u3_httd* htd_u = c3_calloc(sizeof(*htd_u));
+  htd_u->sax_p = u3h_new();
+  htd_u->nax_p = u3h_new_cache(1000); // XX what size?
```

Comment:
should this be configurable? Regardless, we'll want a reasonable default; not sure how to determine that.

Reply:
I'd pick a power of 2 for this number, but 1000 strikes me as vaguely reasonable, if a little on the high side.  512 would be fine given our current use case, I think.  Limiting total memory use would probably be better, but that would require a lot more work.

Reply:
Configurability would be good, but not crucial in this first phase.